### PR TITLE
Fix scale for gyro readings

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight-configurator/issues/3072

### CODE REVIEW

**Comment by KarateBrot** [comment]
Works great

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
